WcaOnRails/db/migrate/20190825095512_add_fields_to_competitions_table.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddFieldsToCompetitionsTable < ActiveRecord::Migration[5.2]
+  def change
+    add_column :Competitions, :announced_by, :integer, null: true, default: nil
+    add_column :Competitions, :results_posted_by, :integer, null: true, default: nil
+    add_column :Competitions, :main_event, :string, null: true, default: nil
```

Comment:
nit: I think we tend to suffix with `_id` in the other places in our database (such as `event_id` in the `competition_events` table), let's do that here for consistency, so `main_event_id` instead.

Reply:
Makes sense, I'm updating this.